configs/train_config.json

```diff
@@ -162,34 +162,36 @@
   },
   "agent_4": {
     "type": "ppo",
+    "prefix": "ppo_input",
     "actor": {
       "type": "tower_actor",
+      "embedder": {
+        "type": "shared_embedding_encoder"
+      },
       "ff_net": {
         "type": "tower",
         "input_dim": 20,
         "hidden_dims": [
           64,
           64
         ],
-        "output_dim": 1,
-        "output_prefix": "actor_emb"
```

Comment:
It is needed because you want to put the output of the actor in the `inputs` dict. The same is needed for critic